2017/ru/0x00-toc.md

```diff
@@ -0,0 +1,28 @@
+# Содержание
+
+|                                                                   |      |      |
+| :---------------------------------------------------------------- | :--- | ---: |
+| С	-	Об OWASP                                                    |      |    1 |
```

Comment:
Also, I think it makes sense to use relative links to corresponding docs here:

```markdown
| С	-	Об OWASP    |      |    [1](0x01-about-owasp.md) |
```

Reply:
Done.